plugins/fmeflow-api-client-node/src/index.ts

```diff
@@ -57,4 +62,46 @@ export class FMEFlowClient {
     const json = await res.json();
     return json.items ?? [];
   }
+
+  async fetchCompletedJobs(): Promise<CompletedWorkspaceJob[]> {
+    const url = `${this.baseUrl}/transformations/jobs/completed?completedState=success&limit=100&offset=0&repository=${this.repository}`;
+
+    const res = await fetch(url, {
+      headers: {
+        Accept: 'application/json',
+        ...(this.token && { Authorization: `fmetoken token=${this.token}` }),
+      },
```

Comment:
This pattern is repeated for every function here, maybe replace by a method so that each calling site just becomes

```suggestion
      headers: this._fetchHeaders(),
```